src/server/middleware/contenttrust/contentrust.go

```diff
@@ -44,42 +43,56 @@ func Cosign() func(http.Handler) http.Handler {
 			return err
 		}
 
-		// If cosign policy enabled, it has to at least have one cosign signature.
+		// If signature policy enabled, it has to at least have one cosign signature.
```

Comment:
If signature policy enabled, it has to at least have one `cosign signature`. 
The `cosign signature` statement here in the comment seems not correct/accurate.